src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -687,14 +687,19 @@ def __init__(self, base_ring, a, b, names='i,j,k'):
         self._gens = (self([0, 1, 0, 0]), self([0, 0, 1, 0]), self([0, 0, 0, 1]))
 
     @cached_method
-    def maximal_order(self, take_shortcuts=True):
+    def maximal_order(self, order_basis=None, take_shortcuts=True):
```

Comment:
This is a backwards incompatible change. I would instead put `order_basis` last because of this.